Interview/demo.md

```diff
@@ -63,7 +63,7 @@ The following requirements should be met:
 
 * Use Swift 4.0 or above.
 * The information (posts and post details) should be available offline. It's assumed that, if it's the first time you are accessing the app and you are offline, you shouldn't see any data.
-* The code should be production grade.
```

Comment:
I disagree with the removal of this requirement. It's not obvious to a lot of candidates.

Reply:
Reverted in 1f55082.